sc-platform-api/src/api/article/article.controller.ts

```diff
@@ -25,4 +26,16 @@ export class ArticleController {
   ): Promise<ArticleResponse> {
     return this.articleService.create(body, authorId);
   }
+
+  @ApiEndpoint({
+    summary: 'Get an article with selected id',
+    interceptors: MapInterceptor(Article, ArticleResponse),
+    documentation: ArticleDocumentation.GET,
```

Comment:
The name of property in documentation object must be the same as method's name in controller
```suggestion
    documentation: ArticleDocumentation.GET_ARTICLE,
```